### DISCUSSION TITLE
How to ignore sidercar's iPad

### DISCUSSION THREAD

**Discussion by micturkey** [MonitorControl/MonitorControl#969] [Q&A]
Hi, when I am using monitorcontrol when I am not connecting to a real external display but iPad in sidecar, I find something strange.
My iPad always dims after a few seconds when sidecar is enabled and when I quit MonitorControl it returns to normal.
I have already disabled sidecar display brightness adjusting in MonitorControl but I think MonitorControl is still controlling iPad's brightness. Thus I want to know how to completely ignore iPad in MonitorControl.
Could anyone help me? Thanks!

**Comment by waydabber** [comment]
Hi - is the Sidecar brightness at 100%? But you are right, there should be a setting to ignore displays or override control method.

**Reply by micturkey** [reply]
In the display configuration, I have unchecked the checkbox of adjust brightness. So I cannot see the brigntness😂
I also find something that if I uncheck the option "sync brightness between apple and builtin.....", it seems that iPad wont't dim any more.
You know that is a good option when I connect to external display.
Therefore, if there is an option completely ingoring some display, it will be better.

**Reply by micturkey** [reply]
And I will turn on the option "sync brightness between apple and builtin.....", and turn on the brightness adjust option to see if the brightness of iPad is at 100% or not.

**Reply by waydabber** [reply]
Yeah, syncing can change the brightness, probably the added light coming from the sidecar display made the macbook decrease its light a bit through the amibent light sensor which was replicated to the sidecar. You should increase the built-in display's brightness if you want these to synced up. A future enhancement might be a setting to define which displays are included in brightness syncing.

**Reply by micturkey** [reply]
Thank you for your reply!
After my experiment, I found that it is really syncing which change the brightness of my sidecar iPad and it seems that syncing is not affected no matter whether the display configuration's brightness checkbox is on or off.
A setting to define which displays are included in syncing will be useful! Looking forward to that function :)

**Reply by riobard** [reply]
@waydabber I'm running into similar issue: Sidecar display is software dimmed even when I explicitly disable brightness control for it in the advanced settings.
